RELEASING.md

```diff
@@ -1,9 +1,14 @@
 # Exposure Notifications Servers Release Process
 
-  * Releases are cut every Tuesday (target time 9am, US Pacific Time)
+  * Releases are cut every other 
+    * Monday for this server (target time 9am, US Pacific Time)
```

Comment:
Is there a reason we don't do them at the same time? I'd rather lump that toil into a single task:

- Relnotes key server, tag release
- Update deps in verification service
- Relnotes verification server, tag release